winlogbeat/etc/winlogbeat.template.json

```diff
@@ -24,6 +24,10 @@
         "@timestamp": {
           "type": "date"
         },
+        "event_id": {
```

Comment:
It looks like we both forgot to run the script to generate the template :) We only generated the docs.